components/CommonAllMembers/index.tsx

```diff
@@ -31,6 +31,7 @@ const CommonAllMembers = ({navigation, chatroomID, isDM}: any) => {
   const dispatch = useAppDispatch();
 
   const {user, community} = useAppSelector(state => state.homefeed);
+  let memberState = user?.state;
```

Comment:
why?

Reply:
To support both flow in DM, member and CM. Member has state '4' and CM has '1'